docs/desktop-wpf/fundamentals/styles-templates-create-apply-template.md

```diff
@@ -0,0 +1,183 @@
+author: thraka
+ms.author: adegeo
+ms.date: 10/20/2019
+dev_langs: ["csharp", "vb"]
+helpviewer_keywords: 
+  - "control contract [WPF]"
+  - "controls [WPF], visual structure changes"
+  - "ControlTemplate [WPF], customizing for existing controls"
+  - "skinning controls [WPF]"
+  - "controls [WPF], appearance specified by state"
+  - "templates [WPF], custom for existing controls"
+ms.assetid: 678dd116-43a2-4b8c-82b5-6b826f126e31
+---
+
+# Create a template for a control
+
+With Windows Presentation Foundation (WPF), you can customize an existing control's visual structure and behavior with your own reusable template. Templates can be applied globally to your application, windows and pages, or directly to controls. Most scenarios that require you to create a new control can be covered by instead creating a new template for an existing control.
+
+[!INCLUDE [desktop guide under construction](../../../includes/desktop-guide-preview-note.md)]
+
+In this article, you'll explore creating a new <xref:System.Windows.Controls.ControlTemplate> for the <xref:System.Windows.Controls.Button> control.
+
+## When to create a ControlTemplate
+
+Controls have many properties, such as <xref:System.Windows.Controls.Border.Background%2A>, <xref:System.Windows.Controls.Control.Foreground%2A>, and <xref:System.Windows.Controls.Control.FontFamily%2A>. These properties control different aspects of the control's appearance, but the changes that you can make by setting these properties are limited. For example, you can set the <xref:System.Windows.Controls.Control.Foreground%2A> property to blue and <xref:System.Windows.Controls.Control.FontStyle%2A> to italic on a <xref:System.Windows.Controls.CheckBox>. You create a <xref:System.Windows.Controls.ControlTemplate> when you want to customize the control's appearance beyond what setting the other properties on the control will do.
+
+In most user interfaces, a button has the same general appearance: a rectangle with some text. If you wanted to create a rounded button, you normally have to create a new control that inherits from the button or recreates the functionality of the button. Your new user control would then provide that circular visual.
+
+You can avoid creating new controls by customizing the visual layout of an existing control. With a rounded button, you create a <xref:System.Windows.Controls.ControlTemplate> with the wanted visual layout.
+
+On the other hand, if you need a control with new functionality, different properties, and new settings, you would create a new <xref:System.Windows.Controls.UserControl>.
+
+## Prerequisites
+
+Create a new WPF application and in *MainWindow.xaml* (or another window of your choice) set the following properties on the `<Window>` element:
+
+|     |     |
+| --- | --- |
+| **Title**         | `Template Intro Sample` |
+| **SizetoContent** | `WidthAndHeight` |
+| **MinWidth**      | `250` |
+
+Set the content of the `<Window>` element to the following XAML:
```

Comment:
I won't correct every occurrence, but style guide says to format XML elements as bold with no angle brackets.